packages/web3-circle-sdk/src/types.ts

```diff
@@ -1098,15 +1098,15 @@ export type Wallet = {
   /**
    * Unique system generated identifier for the user.
    */
-  userId: string;
+  userId?: string;
   /**
    * System-generated unique identifier of the resource.
    */
   walletSetId: string;
   /**
    * For NEAR blockchains only, the originally assigned public key of a wallet at the time of its creation.
    */
-  initialPublicKey: string;
+  initialPublicKey?: string;
```

Comment:
these are optional and not always returned, the type must show that. also not all are strings but fixed set of values (like `EOA | SCA`), could be also indicated by the type